src/services/chat_service.py

```
@@ -0,0 +1,154 @@
+        # LLM 생성 파라미터 설정
+        config = GenerationConfig(
+            temperature=0.0,  # temperature : 0에 가까울수록 가장 확률 높은 단어를 선택하여 답변이 논리적이고 정해진 틀을 따릅니다.
+            max_output_tokens=1024,  # max_output_tokens : 생성할 최대 토큰 수 (약 700~800단어 분량)
+            top_k=20,  # top_k : 상위 20개 단어만 후보로 둠 (이상한 단어 차단)
+            top_p=0.7,  # top_p : 상위 70% 확률 내에서만 선택 (신뢰도 확보)
+        )
+        response = generative_model.generate_content(prompt, generation_config=config)
+        final_answer = response.text
+    except Exception as e:
+        print(f"❌ Gemini 호출 실패: {e}")
+        raise e  # 에러를 다시 던져서 API 엔드포인트에서 처리하도록 함
```

Ruff (0.14.8)

Use `raise` without specifying exception name

Remove exception name

(TRY201)